DigitalOffice/Services/CompanyService/Models/Position.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.CompanyService.Models
+{
+    public class Position
+    {
+        public string Name { get; set; }
+        public string Description { get; set; }
+        public IEnumerable<Guid> UserIds { get; set; }
```

Comment:
IEnumerable?

Reply:
See PR with GetPositionsList and AddPosition endpoints

Reply:
ToList

Reply:
Fixed